specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -5331,6 +5450,135 @@
         }
       }
     },
+    "IpAddressAggregate": {
+      "description": "The key value pair for properties.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique ID for the entree",
+          "type": "string"
+        },
+        "tenantId": {
+          "description": "The tenant ID",
+          "type": "string"
+        },
+        "serviceId": {
+          "description": "The service ID",
+          "type": "string"
+        },
+        "ipAddress": {
+          "description": "The Bad Ip Address",
+          "type": "string"
+        },
+        "timestamp": {
+          "description": "When the event occurred",
+          "type": "string"
+        },
+        "firstAuditTimestamp": {
+          "description": "The first audit timestamp",
+          "type": "string"
+        },
+        "lastAuditTimestamp": {
+          "description": "The last audit timestamp",
+          "type": "string"
+        },
+        "extranetLockoutErrorCount": {
+          "description": "The extranet lockout error count",
+          "type": "integer"
+        },
+        "badPasswordErrorCount": {
+          "description": "The bad password error count",
+          "type": "integer"
+        },
+        "uniqueUsernamesAttemptedCount": {
+          "description": "The unique usernames attempted",
+          "type": "integer"
+        },
+        "attemptCountThresholdIsExceeded": {
+          "description": "Has the attempt count threshold been exceeded",
+          "type": "boolean"
+        },
+        "timeSpan": {
+          "description": "The timespan of the event",
+          "type": "string"
+        },
+        "isWhitelistedIpAddress": {
+          "description": "Is the address a whitelisted IP address?",
+          "type": "boolean"
+        },
+        "networkLocation": {
+          "description": "The network location",
+          "type": "string"
+        },
+        "attemptCountThresholdOnTrigger": {
+          "description": "The attempted count threshold on trigger.",
+          "type": "integer"
+        },
+        "attemptThresholdTypeOnTrigger": {
+          "description": "The attempt threshold type on trigger.",
+          "type": "string"
+        },
+        "geographicLocation": {
+          "description": "The geographic location.",
+          "type": "string"
+        }
+      }
+    },
+    "IpAddressAggregates": {
+      "description": "The list of Risky IP Aggregates.",
```

Comment:
nit: please standardize Risky, Bad etc to a single term to describe "bad" IP aggregates. Also if turns into two words, should it be IP **address a**ggregates?

Reply:
Fixed